salt/personalised-covers/config/srv-personalised-covers-config-config.php

```diff
@@ -1,13 +1,8 @@
 <?php
 
 return [
-	{% if pillar.elife.env in ['ci', 'dev']  %}
+	{% if pillar.elife.env not in ['prod', 'end2end', 'continuumtest']  %}
 	'debug' => true,
-	'aws_credentials' => [
```

Comment:
potentially controversial change here. from my test instance by removing these the app began working again and the smoke tests completed.

I can see how they would be useful during development and how they might have hung around afterwards, but I'm not sure how per-env infrastructure (iam and s3 for ci, end2end, prod) could have been configured and this never touched. Perhaps because the smoke tests weren't enabled or it was only tested on end2end or prod ...?

anyway. I'm not sure how correct it is.

Reply:
We need this to stay. `dev` and `ci` shouldn't talk to a real bucket as it's too difficult to reset between test runs. The idea is that you should be able to run the tests of a random, broken pull request and the next one shouldn't be affected.

The project is then correctly configured to talk to a `fakes3` local simulator of S3 buckets, but there is no trace in the formulas of actually running `fakes3` in `personalised-covers--ci`.